Application/components/FriendsColumn/FriendsTab.tsx

```diff
@@ -48,6 +52,18 @@ export function FriendsTab({senderUsername, senderID, privateChat, onMessageExch
     const [searchQuery, setSearchQuery] = useState<string>('');
     const [receiverUsername, setreceiverUsername] = useState<string>('');
     const [receiverID, setReceiverID] = useState<string>('');
+    const { updateContext, setActiveView } = useChat();
+    
+    const { channel } = useChannel(getSystemsChannelID(), (message) => {
```

Comment:
- Consider implementing error handling mechanisms for API requests, especially in the useChannel hook callback. If an error occurs while subscribing to the Ably channel, users might not receive updates for friend requests or other real-time events.